src/http/fetch_request.ts

```diff
@@ -156,8 +156,8 @@ export class FetchRequest {
     }
   }
 
-  get isIdempotent() {
-    return this.method == FetchMethod.get
+  get isSafe() {
+    return this.method === FetchMethod.get
```

Comment:
As pointed by MDN, `HEAD` and `OPTIONS` are safe too. Do we want to include them here? Don't think we're using them, but they are part of the fetch API.

Reply:
I thought about that, but opted to leave it as-is since we [don't define them](https://github.com/hotwired/turbo/blob/060dd7afde467d718dd1afb4ed450026402388d0/src/http/fetch_request.ts#L30-L36). Adding them to the enum suggests (perhaps) that we are using them somewhere,

Reply:
Fair enough 👍